pkgs/applications/science/robotics/mujoco/default.nix

```diff
@@ -129,15 +129,15 @@ let
 
 in stdenv.mkDerivation rec {
   pname = "mujoco";
-  version = "3.1.0";
+  version = "3.1.1";
 
   # Bumping version? Make sure to look though the MuJoCo's commit
   # history for bumped dependency pins!
   src = fetchFromGitHub {
     owner = "google-deepmind";
-    repo = pname;
-    rev = version;
-    hash = "sha256-a8h30psoAlj9dI4j8IfY3WzWjY4MrRosGbvgt79s1BQ=";
+    repo = "mujoco";
+    rev = "refs/tags/${version}";
```

Comment:
Does this pull in something else?

Reply:
No, I have just been told to explicitly set `refs/tags` in `rev` because it was more reliable.

Reply:
`refs/tags` is just more verbose. The more common pattern I see is to use `repo = version; repo = pname`.

Reply:
I have also heard the `refs/tags` suggestion before, but AFAIK we don't have this written down in any established place yet. This seems like a good opportunity to advocate for the creation of a nixpkgs style guide to adjudicate on these sorts of things

in terms of this pr I don't have a preference either way

Reply:
I guess that it was precisely the goal of https://github.com/NixOS/nixpkgs/pull/257099